aptos-move/aptos-release-builder/src/components/mod.rs

```diff
@@ -172,6 +183,22 @@ impl ReleaseEntry {
                 let gas_schedule = gas_override_default(gas_overrides)?;
                 if !fetch_and_equals::<GasScheduleV2>(client, &gas_schedule)? {
                     result.append(&mut gas::generate_gas_upgrade_proposal(
+                        true,
+                        &gas_schedule,
+                        is_testnet,
+                        if is_multi_step {
+                            get_execution_hash(result)
+                        } else {
+                            "".to_owned().into_bytes()
+                        },
+                    )?);
+                }
+            },
+            ReleaseEntry::DefaultGasWithOverrideOld(gas_overrides) => {
+                let gas_schedule = gas_override_default(gas_overrides)?;
+                if !fetch_and_equals::<GasScheduleV2>(client, &gas_schedule)? {
+                    result.append(&mut gas::generate_gas_upgrade_proposal(
```

Comment:
do we actually need this? should we just pass the flag based on whether randomness is enabled on-chain?

though I am fine with explicit way as it is now

Reply:
we want to check the on-chain framework version then. is framework version a thing?

Reply:
argh. I don't think that's possible